example/javascriptlib/module/1-common-config/build.mill

```diff
@@ -0,0 +1,67 @@
+package build
+
+import mill._
+import mill.javascriptlib._
+
+object foo extends TypeScriptModule {
+
+  def customSource = Task {
+    Seq(PathRef(millSourcePath / "custom-src" / "foo2.ts"))
+  }
+
+  def allSources = Task.Anon {
+    super.allSources() ++ customSource()
+  }
+
+  def resources = Task.Sources {
+    super.resources() ++ Seq(PathRef(millSourcePath / "custom-resources"))
+  }
+
+  def generatedSources = Task {
+    for (name <- Seq("A", "B", "C")) os.write(
+      Task.dest / s"foo-$name.ts",
+      s"""export default class Foo$name {
+         |  static value: string = "Hello $name"
+         |}
+      """.stripMargin
+    )
+
+    Seq(PathRef(Task.dest))
+  }
+
+  def mkENV = Task {
```

Comment:
Let's rename this `def forkEnv` for consistency with the Java/Scala/Kotlin equivalent